modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/test/js/components/list/Item.es.js

```diff
@@ -122,4 +123,12 @@ describe('Item', () => {
 
 		expect(onFocusFn.mock.calls.length).toBe(0);
 	});
+
+	it('renders a link to viewURL', () => {
+		const viewURL = 'http://www.google.com';
```

Comment:
Can we change this to a dummy URL like `#`? or at least liferay.com rather than google.